src/state_manager/mod.rs

```diff
@@ -1726,6 +1758,46 @@ where
 
         Ok((state_root, invoc_trace))
     }
+
+    /// Attempts to lookup the state and receipt root of the next tipset.
+    /// This is a performance optimization to avoid recomputing the state and receipt root by checking the blockstore.
+    /// It only checks the immediate next epoch, as this is the most likely place to find a child.
+    fn try_lookup_state_from_next_tipset(&self, tipset: &Tipset) -> Option<StateOutputValue> {
+        let epoch = tipset.epoch();
+        let next_epoch = epoch.saturating_add(1);
```

Comment:
It should be fine to use regular `+` operator. If we ran out of epochs in that type, we have much worse problems.

Reply:
Done.